cape_encrypt/cape_encrypt/cape_encrypt.py

```diff
@@ -0,0 +1,79 @@
+"""
+Cape Encrypt functionality for use within functions deployed to run in a Cape Enclave.
+
+This provides the ability to be able to encrypt or decrypt information solely within a
+Cape Enclave using the Cape Key associated with the function owner. Functions can then
+encrypt data generated during the execution of the function or have fine grained
+control over decrypting inputs.
+"""
+import json
+import socket
+
+
+def connect() -> socket.socket:
```

Comment:
```suggestion
def _connect() -> socket.socket:
```
i assume this function is not meant to be part of the public API? if so, the leading underscore signals that it's meant as a private function for this module (even though python does nothing to enforce that, it's considered a best practice)